src/store/index.ts

```diff
@@ -166,6 +168,7 @@ export const Store = createStore<IRootState>({
             APP_NAME: packageInfo.productName,
             APP_VERSION: packageInfo.version,
             APP_VERSION_TAG: versionInfo["version-tag"],
+            SDK_VERSION_TAG: Vircadia.verboseVersion ?? "probably 0.0.4",
```

Comment:
Nani?

Reply:
this is a temp thing. The text string should never display. It's for only down rev SDK versions.